### DISCUSSION TITLE
tagged template string WTF

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#371] [Dicas]
Ok, abra o seu Chrome ou Firefox

Defina uma função qualquer
```js
const wtf = (...args) => { console.log(args) }

// Agora, na parte esquerda de um string literal, coloque a função
wtf`JS${1} teste ${2} `
```

Firefox 47
![screenshot from 2016-11-22 18-00-22](https://cloud.githubusercontent.com/assets/736728/20539810/97da2f4c-b0dd-11e6-8c43-05d59cf96171.png)

Chrome 54
![screenshot from 2016-11-22 18-01-58](https://cloud.githubusercontent.com/assets/736728/20539860/c9df7dd0-b0dd-11e6-95e1-31730f9a002f.png)

[have fun!](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_template_strings) :hankey: estou chocado com este aprendizado haha

Quais outras funções do JS você consegue chamar sem usar os (parênteses)?

**Comment by MarcoWorms** [comment]
```javascript
throw 'poop'
```

**Comment by leandroruel** [comment]
:open_mouth:
